elements/pfe-card/pfe-card.ts

```diff
@@ -55,6 +55,10 @@ export class PfeCard extends LitElement {
 
   static readonly styles = [style];
 
+  @colorContextProvider() color?: Color;
```

Comment:
discussion: is it worth the boilerplate / bundle size cost to require these to be explicitly defined?

or is it enough to declare them in typescript / jsdoc comments?

Reply:
I *think* I would prefer them to be explicitly defined.

Reply:
Yeah, thinking it over this makes sense, @heyMP, so i did it.

One good indication that this is the right choice is that making these attributes explicit raised a dilemma that wasn't obvious previously:

In order to pass the unit tests, I had to set an initializer for both **card** and **band**'s `colorPalette` properties, defaulting to `'base'`. This essentially overrules it's context every time. in other words, since card and band always start with `color-palette="light"`, their `on` attributes are basically irrelevant.

You can see this in action on the [accordion demo](https://patternflyelements.org/components/accordion/demo/?accordion-with-history=2) if you open the "nested 2 card layout" accordion. Run this snippet in the devtools console, and you'll see that the cards retain their initial 'base' color palette despite the surrounding context being 'dark':

```js
document.getElementById('context-band').setAttribute('color', 'darkest')
```

(remember, this is on main branch, so what you're seeing there passed unit tests)

@coreyvickery is this intended? Should card and band always 'reset' their context unless explicitly provided with a palette (i.e. `darker`, `accent`, etc), or would we like card (and band) to respect their ancestors' color context?

Reply:
@bennypowers I feel like resetting would be best.